src/Autofac.Integration.WebApi.Owin/Autofac.Integration.WebApi.Owin.nuspec

```diff
@@ -14,8 +14,8 @@
     <iconUrl>https://cloud.githubusercontent.com/assets/1156571/13684110/16b8f152-e6bf-11e5-84ae-22c66c6d351a.png</iconUrl>
     <releaseNotes>Release notes are at https://github.com/autofac/Autofac.WebApi.Owin/releases</releaseNotes>
     <dependencies>
-      <dependency id="Autofac" version="[3.5.0,5.0.0)" />
-      <dependency id="Autofac.WebApi2" version="[3.4.0,5.0.0)" />
+      <dependency id="Autofac" version="[5.0.0,6.0.0)" />
+      <dependency id="Autofac.WebApi2" version="[5.0.0,6.0.0)" />
       <dependency id="Autofac.Owin" version="[4.0.0,5.0.0)" />
```

Comment:
```suggestion
       <dependency id="Autofac.Owin" version="[5.0.1,6.0.0)" />
```

Reply:
Done; again, thanks for catching it. Should have triple-checked it!